src/lib/components/icons/SvgIcon/SvgIcon.types.ts

```diff
@@ -5,5 +5,6 @@ export type IIconSize = 'S' | 'M' | 'L' | 'XL';
 export type ISvgIconProps = PropsWithChildren<
   SVGAttributes<any> & {
     size?: IIconSize;
+    isActive?: boolean;
```

Comment:
why you adding the isActive props to all SvgIcon Components? do we use it in another place?
think how you can add it only for this component....